backend-elasticsearch/src/main/java/org/hibernate/search/v6poc/backend/elasticsearch/search/predicate/impl/BooleanJunctionPredicateBuilderImpl.java

```diff
@@ -43,6 +57,34 @@ public ElasticsearchSearchPredicateCollector getFilterCollector() {
 		return this::filter;
 	}
 
+	@Override
+	public void minimumShouldMatchNumber(int constraintIgnoreCeiling, int matchingClausesNumber) {
```

Comment:
Thinking about a bit more as you need a term for the log message, maybe "ignoreConstraintCeiling" would be better?

Reply:
Maybe we should ask someone native if he has a better idea on how to convey the message we want to convey.

Not sure there's a good solution as I think the issue lies in the choice of Solr to not make it a simple threshold.